### PR TITLE
Add functionality for calculator

### PR DESCRIPTION
**Pull request by ammarkandel**
# Add full functionality for the calculator.
## Add operations and calculation.

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @ammarkandel, congratulations on this project :fireworks: The right linter files were used :+1: The calculate and operate logic works as expected. To get our project approved :slightly_smiling_face: Let us kindly address the comments below:

- Kindly add the commands in cloning the repo so that non-technical people can also run the app; for example `git clone "repo"`
- Kindly wrap the second number with the Big.js class

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import Big from 'big.js'; // eslint-disable-line
+
+const operate = (num1, num2, operation) => {
+  const n = Big(num1);
+  let evaluation;
```
Comment (uche-inyama) [inline src/logic/operate.js]: Kindly add this code snippet after line 4.

``` const n2 = Big(num2) ```

Then replace num2 inside operate function with n2.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi @ammarkandel :wave: 

Like the last reviewer pointed out, pass **num2** as an argument to the Big instance, just like you did to **num 1**

#### That's it, make this change the submit another form for code review.



### Happy coding :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer :wave:

Thank You for submitting this Pull Request. I am thrilled to inform you that your project is approved now. :clap:

You have done a great job. I like your project. 




In short, I am very excited to see this project and proud of you. 

**What to do next?** :thinking:

1- Visit https://dashboard.microverse.org/projects and submit your approved project. 
2- Prepare yourself for the next project. 

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**